sdk/core/core-http/test/credentialTests.ts

```diff
@@ -87,21 +87,21 @@ describe("Basic Authentication credentials", () => {
       it("should fail with options.inHeader and options.inQuery set to null or undefined", function (done) {
         (function () {
           new ApiKeyCredentials({ inHeader: undefined, inQuery: undefined } as any);
-        }).should.throw();
+        }.should.throw());
```

Comment:
nitpick, looks like unnecessary changes, could be reverted?

Reply:
Pipelines was failing due to a formatting issue, [https://dev.azure.com/azure-sdk/public/_build/results?buildId=2133687&view=logs&j=b70e5e73-bbb6-5567-0939-8415943fadb9&t=70ccc7c9-8d04-5255-456e-9568fe142bcd&l=115](url), is there any other way to resolve this, I ran rushx format in core-http as suggested.